docs/WebsocketAPIV4.md

```diff
@@ -0,0 +1,235 @@
+## WebSocket protocol v4 details
+
+WebSocket Protocol v4 requires an Exasol version of at least 8.31.0. It follows the RFC 6455 document of the IETF.
+
+The Exasol connection server identifies the initial GET request by the client.
+This request contains information about the used protocol version.
+Depending on this information the matching login and protocol class is
+chosen.
+
+After the handshake the process is identical to a connection using the
+standard drivers like JDBC or ODBC: The connection server listens to
+incoming messages and forwards the requests to the database. 
+
+## Table of contents
+* [Changes](#changes)
+* [Command summary](#command-summary)
+* [Attributes: Session and database properties](#attributes-session-and-database-properties)
+* [Data Types: Type names and properties](#data-types-type-names-and-properties)
+* [Compression](#compression)
+* [Heartbeat/Feedback messages](#heartbeatfeedback-messages)
+* [Subconnections](#subconnections)
+
+## Changes
+
+| Date | Exasol Version | Change | Issue |
+| --- | --- | --- | --- |
+| 2024.09.05 | 8.31.0 | The latest update to Exasol enables the use of the database port for subconnections instead of requiring ports in the range 20000 to 21000. Additionally, this update eliminates the need for special IP address configurations on the database. | |
+| 2024.09.05 | 8.31.0 | The `enterParallel` command has been deprecated. | |
+
+## Command summary
+
+### Connection-related commands
+
+The following commands are used to connect to Exasol, disconnect from Exasol, 
+and query the hosts of an Exasol cluster.
+
+| Command | Description |
+| --- | --- |
+| [disconnect](commands/disconnectV1.md) | Closes a connection to Exasol |
+| [enterParallel](commands/enterParallelV1.md) | Opens subconnections for parallel execution<br>:warning: Deprecated, please use [requestParallelConnections](commands/requestParallelConnectionsV4.md) instead. |
```

Comment:
Can we remove it here?

Reply:
It is still usable, isn't it? Or was it removed from v4?

Reply:
It looks like it's still available in v4.

Reply:
Yes. It can be used in v4. I taught as it is deprecated we can remove it from v4.